frontend/packages/eslint-plugin-console/lib/config/prettier.js

```diff
@@ -0,0 +1,49 @@
+const merge = require('merge');
+
+module.exports = {
+  extends: ['prettier', 'prettier/react'],
+
+  plugins: ['prettier'],
+
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true,
+        trailingComma: 'all',
+        bracketSpacing: true,
+        jsxBracketSameLine: false,
+        arrowParens: 'always',
+        printWidth: 120,
+        semi: true,
+        useTabs: false,
+      },
+      {
+        usePrettierrc: false,
+      },
+    ],
+  },
+
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: merge(require('./rules/prettier-typescript'), {
+        'prettier/prettier': [
+          'error',
+          {
```

Comment:
I am noticing that this does not match the `.prettierrc` configuration. Linting works correctly in this case as we provide `usePrettierrc: false` here, but editors such as vscode format code based on `.prettierrc` file unless user explicitly disables this in editor configuration and updates editor settings to match our config. I don't think this is desirable.

So IMHO we should align .prettierrc to the configuration we introduce here.

Reply:
Ah, please ignore this one, error was in my local prettierrc... But still, should we at this point introduce the project-wide .prettierrc file?

Reply:
My thought was that we prefer using eslint over prettierrc so all rules are in one place.
If there are files that aren't linted that we still want to run prettier for than I guess it makes sense to use the RC file.

I'm for whatever gets us clean formatting across the project :)

Though right now we don't want to force the formatting on some fitted that haven't already been swept.

Reply:
I found out that my vscode's 'prettier.eslintIntegration' was set to false and did not get picked up from .vscode/settings.json which caused it to ignore the prettier configuration from eslint. So yeah, as long as 'prettier.eslintIntegration' is set to true, .prettierrc is not needed.